test/fixtures/transaction_builder.json

```diff
@@ -723,11 +723,11 @@
             "vout": 1,
             "signs": [
               {
-                "privKey": "KwDiBf89QgGbjEhKnhXJuH7LrciVrZi3qYjgd9M7rFU73sVHnoWn",
+                "keyPair": "5HpHagT65TZzG1PH3CSu63k8DbpvD8s5ip4nEB3kEsreAnchuDf",
```

Comment:
These are the same private keys, just different compression flags (they were wrong previously)